index.html

```
@@ -0,0 +1,4 @@
+<h1>This is my heading</h1>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)